lib/github-pages-health-check/cloudflare.rb

```diff
@@ -1,9 +1,10 @@
-class GitHubPages
-  class HealthCheck
+module GitHubPages
```

Comment:
True. I guess if you don't think there's ever a reason you should have to instantiate `GitHubPages`, then it should be a module. :)